pkg/serverless/trace/span_modifier.go

```diff
@@ -16,8 +16,9 @@ const (
 )
 
 type spanModifier struct {
-	tags                 map[string]string
-	coldStartSpanCreator *ColdStartSpanCreator
+	tags            map[string]string
+	lambdaSpanChan  chan *pb.Span
+	ColdStartSpanId uint64
```

Comment:
This is a total nitpick. I don't think this field needs to be exported since the `spanModifier` type itself isn't exported.